docs/modules/configuration/pages/connections.adoc

```diff
@@ -1,7 +1,158 @@
 = Connections
 :imagesdir: ../assets/images
 
-First of all, we have to define the connections the plugin uses to connect to the Orchestrator.
+First, we have to define the connections the plugin uses to connect to the Orchestrator. A connection consists of
+an orchestrator URL, an API key to authenticate against the orchestrator, and an alias used to reference it. Connections
+are saved internally in the Secure Credentials Vault.
+
+Connections can be added, validated, and modified via both ReST endpoints and karaf shell commands.
+
+== ReST
+
+NOTE: By default, the plugin will validate new connections before saving them. This can be overridden in the appropriate endpoints with the *force* query parameter.
+
+The plugin mounts all endpoints at the following location.
+```
+http://localhost:8980/opennms/rest/velocloud/
+```
+
+=== GETs (Retrieving Data)
+[options="header", cols="5,10"]
```

Comment:
```suggestion
[options="header, autowidth" cols="1,2"]
```
We have a different style for columns, which is to put everything on separate lines  (see feedback below). Please verify it still renders correctly.